app/javascript/lib/honeycrisp.js

```diff
@@ -98,28 +106,32 @@ var followUpQuestion = (function() {
 
                 // set initial state of follow-ups based on the page
                 $(this).find('input').each(function(index, input) {
-                    if($(this).is(':checked') && $(this).attr('data-follow-up') != null) {
-                        $($(this).attr('data-follow-up')).show();
+                    if($(this).attr('data-follow-up') != null) {
+                        $($(this).attr('data-follow-up')).toggle($(this).is(':checked'));
                     }
                 });
 
                 // add click listeners to initial question inputs
-                $(self).find('> .question-with-follow-up__question input').click(function(e) {
-                    // reset follow ups
-                    $(self).find('> .question-with-follow-up__follow-up input').attr('checked', false);
-                    $(self).find('> .question-with-follow-up__follow-up').find('.radio-button, .checkbox').removeClass('is-selected');
-                    $(self).find('> .question-with-follow-up__follow-up').hide();
-
-                    // show the current follow up
-                    if($(this).is(':checked') && $(this).attr('data-follow-up') != null) {
-                        $($(this).attr('data-follow-up')).show();
-                    }
-                })
+                $(self).find('.question-with-follow-up__question input').click(function(e) {fUQ.update($(self))})
+            });
+        },
+        update: function ($container){
+            // reset follow ups
+            $container.find('.question-with-follow-up__follow-up input').attr('disabled', true);
+            $container.find('.question-with-follow-up__follow-up').hide();
+
+            $container.find('.question-with-follow-up__question input').each(function(index, input) {
+                // if any of the inputs with a data-follow-up is checked then show the follow-up
+                if($(input).is(':checked') && $(input).attr('data-follow-up') != null) {
+                    $container.find('.question-with-follow-up__follow-up input').attr('disabled', false);
+                    $($(this).attr('data-follow-up')).show();
+                }
             });
         }
     }
     return {
-        init: fUQ.init
+        init: fUQ.init,
+        update: fUQ.update
```

Comment:
this `update` change is a result of https://github.com/codeforamerica/honeycrisp-gem/commit/c05fb0b74c1ecf67a62d9735c5dc77240376beb2